src/commands/platforms/create.ts

```diff
@@ -0,0 +1,166 @@
+import { flags } from '@oclif/command';
```

Comment:
Looks great. Having David product review. Can you add tests for the platform and environment cmds?

Prob worth covering both that CI flow (all values are passed via cmd) and the user flow with inquirer.
https://stackoverflow.com/questions/49862039/how-to-write-unit-tests-for-inquirer-js